pennylane_rigetti/qc.py

```diff
@@ -180,10 +178,8 @@ def define_wire_map(self, wires):
 
     def apply(self, operations, **kwargs):
         """Applies the given quantum operations."""
-        prag = Program(Pragma("INITIAL_REWIRING", ['"PARTIAL"']))
```

Comment:
By default, the compiler will choose `NAIVE` if the program fits the topological constraints of the QPU, and `PARTIAL` otherwise. I've removed this so that we can lean on that behavior instead of overwriting it for every program.